core/ordering/cosipbft/mod.go

```diff
@@ -374,7 +379,13 @@ func (s *Service) main() error {
 			cancel()
 
 			if err != nil {
-				return xerrors.Errorf("round failed: %v", err)
+				if calculateBackoff(backoff+1) < RoundMaxWait {
```

Comment:
Yes because you don't want to reach an infinite timeout but if for instance two many nodes are offline for a long time, it will eventually catch up when they're back online.